docs/source/cli.mdx

```diff
@@ -0,0 +1,48 @@
+# Command Line Interface (CLI)
+
+🤗 Datasets provides a command line interface (CLI) with useful shell commands to interact with your dataset.
+
+You can check the available commands:
+```shell
+$ datasets-cli --help
+usage: datasets-cli <command> [<args>]
+
+positional arguments:
+  {convert,env,test,run_beam,dummy_data,convert_to_parquet}
+                        datasets-cli command helpers
+    convert             Convert a TensorFlow Datasets dataset to a HuggingFace Datasets dataset.
+    env                 Print relevant system environment info.
+    test                Test dataset implementation.
+    run_beam            Run a Beam dataset processing pipeline
+    dummy_data          Generate dummy data.
+    convert_to_parquet  Convert dataset to Parquet
+
+optional arguments:
+  -h, --help            show this help message and exit
+```
+
+## Convert to Parquet
+
+Easily convert your Hub script-dataset to Parquet files, so that the dataset viewer will be supported.
+
+```shell
+$ datasets-cli convert_to_parquet --help
+usage: datasets-cli <command> [<args>] convert_to_parquet [-h] [--token TOKEN] [--revision REVISION] [--trust_remote_code] dataset_id
+
+positional arguments:
+  dataset_id           source dataset ID
```

Comment:
should we precise what a dataset ID is? Here, or inside a small example usage of the CLI below?

Reply:
I feel like people like to copy / edit / paste (I do)

Reply:
I have added an example, because the content here is what the CLI command --help outputs; maybe I could add the example to the CLI command --help as well.

Reply:
Done.